### ISSUE TITLE
Other timezones

### ISSUE THREAD

**Issue by ricsim** [AlexGoodyear/agoodWatch#7]
It seems you can only set the time via NTP.
But it is 2 hours off as I am here in CEST.
How can I change the timezone or manually set the time?

**Comment by ricsim** [comment]
OK ... I noticed there is a define:
#define RTC_TIME_ZONE   "CEST-2"
Now when I synchronize with NTP it shows the correct time.
After the screen times out, it shows completely false time.
Maybe issue with writing the new NTP time into the RTC??

**Comment by careyer** [comment]
Yeah... same problem here. It syncs the wrong time in the first place (17:58 instead of 20:58) and I am living in CEST+1 which is also set in the sketch... and after that when the screen times out the time is completely wrong

**Comment by dbuggz** [comment]
I'm not a good programmer but I've done  a hack that seems more reliable... use a local server, and the configTime() function.


// configTzTime(RTC_TIME_ZONE, "pool.ntp.org");

     // dw 
     // Init and get the time
     long gmtOffset_sec, daylightOffset_sec;
     gmtOffset_sec = -10 * 3600;
     daylightOffset_sec = 0;
 
     configTime(gmtOffset_sec, daylightOffset_sec, "au.pool.ntp.org");
     printLocalTime();
     // dw
